CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Celestia
+
+Welcome and Thank You for contributing to building Celestia, the first modular
+blockchain network. 
+
+In this guide you will find information about how Celestia manages the
+`CelestiaOrg` Github as well as expectation around our Engineering Standards.
+
+## Github Management
+
+The `CelestiaOrg` Github uses teams to manage access to the organization and
+its repositories. Members of the Celestia core team and key contributors are
+added to the Github team that aligns with the scope of work they perform. For
+example, those working on the `celestia-node` repository are added to the
+`celestia-node` team. 
+
+### Permissions
+
+By default, all teams have Triage access to all repositories in the
+`CelestiaOrg` Github. This allows for anyone to help manage issues and pull
+request (i.e. adding labels). Teams are given write access to the repositories
+that they are responsible for working on. For example, the `celestia-node` team
+would have write access to the `celestia-node` repository.
+
+### .github
+
+Celestia utilizes the organization's `.github` repository. This repository is
+used to store common organization level content like Github actions, issue
+templates, PR templates, etc.
+
+## Development
+
+### What the Fork
+
+The default development flow is to fork the repository that you are working on
+in order to submit a PR. If you have write access to a repository, because you
+are a member of that sub team, then you can push your development branches
```

Comment:
[Non Blocking][Question]
Should we specify which one is favorable? dev branches or forks? or leave it to the devs to decide?

Reply:
I think it is fine to be personal preference.